<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addActor</class>
 <widget class="QDialog" name="addActor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="mainLabel">
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ДОБАВЛЕНИЕ АКТЕРА</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="secondNameLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ФАМИЛИЯ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstName">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ИМЯ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="bornLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ДАТА РОЖДЕНИЯ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="born"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="listPerformanceLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>СПИСОК СПЕКТАКЛЕЙ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTableWidget" name="listPerformance">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>230</number>
       </attribute>
       <column>
        <property name="text">
         <string>СПЕКТАКЛЬ</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="createActor">
         <property name="text">
          <string>ДОБАВИТЬ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delActor">
         <property name="text">
          <string>УДАЛИТЬ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="secondName">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelError">
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(240, 0, 0); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCANCEL">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>addActor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCANCEL</sender>
   <signal>clicked()</signal>
   <receiver>addActor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
